src/fosslight_yocto/create_oss_report_for_yocto.py

```
@@ -771,12 +771,12 @@ def run_source_code_analysis_multiprocessing(analyze_all_mode, out_dir, output_f
 
 def run_scancode_per_dir(path_to_scan, json_file_name, num_cores, recipe_name):
     if os.path.isdir(path_to_scan):
-        logger.warning("|- Analyzing: " + recipe_name + ",path:" + path_to_scan + ",json:" + json_file_name)
+        logger.debug("|- Analyzing: " + recipe_name + ",path:" + path_to_scan + ",json:" + json_file_name)
         try:
             rc, results = cli.run_scan(path_to_scan, max_depth=100, strip_root=True, license=True, copyright=True,
```

Ruff (0.15.10)

[warning] Unpacked variable `rc` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)